harbors/management/commands/harbors_connect_to_resources.py

```diff
@@ -16,7 +16,7 @@
     "5141e8c0-3491-46b2-898a-16523c54df48": 13,
     "5299c27c-3cdb-4117-9f69-54bf3576927b": 15,
     "e01a17fc-1702-4a4e-99a7-60ea027b935d": 17,
-    "9190743a-f883-4f92-8923-f572157bf6ab": 16,
+    "9190743a-f883-4f92-8923-f572157bf6ab": 49,
```

Comment:
resources.Harbor.objects.get(pk=9190743a-f883-4f92-8923-f572157bf6ab)
<Harbor: Merihaan venesatama (Hakaniemenranta 13)>

but
harbors.Harbor.objects.get(pk=49)
<Harbor: Merihaan venesatama (Hakaniemenranta 30)>

Is one or the other address wrong in our database? Hakaniemenranta 13 and Hakaniemenranta 30 are on the opposite sides of Merihaka

Reply:
The changes are made on the `resources` fixtures :)